### ISSUE TITLE
An OSS ETF

### ISSUE THREAD

**Issue by defi-er** [dev-protocol/DIPs#25]
Open source software is eating the world. Dev Protocol offers a streamlined service for Github's OSS projects to receive sustainable funding. If Dev Protocol could capture a percentage of each creator token created using Dev Protocol then it would become a backdoor ETF for OSS. DEV holders would receive creator staking rewards + governance power. This would become extremely lucrative for enterprises with dependencies on OSS to vote on the direction of OSS projects. Dev Protocol DAO would hold creator tokens and give voting power proportional to DEV held. This creates a new sink for DEV. The world's first ETF for OSS.

**Comment by aggre** [comment]
Fascinating idea. That OSS ETFs invest in Properties (creator tokens). *This is because APY of staking is common, at the moment.

I think several ETF markets could be built. For example, language-specific ETFs such as JavaScript and Python, and category-specific ETFs such as machine learning and front-end.

A relatively simple script can obtain per-creator-tokens rewards.

**Comment by defi-er** [comment]
Yes, exactly. One large ETF for all internet assets, an ETF for each market, then specific ETFs. These creator tokens will be valuable to those with OSS dependencies. Stakes Social must work very closely with creators to ensure that these creator tokens grant holders voting rights.

**Comment by yumemayu** [comment]
This idea is in line with Dev Protocol's vision of bringing open source innovation to all industries. Dev Protocol has a high flexibility as it started with OSS. We've heard from several VCs about the possibility of investing in property tokens, and ETFs offer possibilities for everyone. An even better thing is that enterprise companies can participate in governance as a stakeholder rather than just sponsoring them.

**Comment by defi-er** [comment]
This would especially be possible if a Dev Protocol entity becomes a fiscal sponsor. We could also explore a partnership with already established fiscal sponsors like OpenCollective or NUMFocus. 

https://osaos.codeforscience.org/whats-a-fiscal-sponosor/

**Comment by defi-er** [comment]
To-Do: Explore and sketch the technical architecture for meta-utility.
